rubocop/rubocop_rails.yml

```diff
@@ -3,11 +3,34 @@ inherit_from: rubocop.yml
 AllCops:
   Exclude:
     - db/schema.rb
+    - 'bin/**/*'
+    - 'config/**/*'
     - 'vendor/**/*'
 
 Rails:
   Enabled: true
 
+Metrics/BlockLength:
+  Exclude:
+    - 'db/seeds.rb'
```

Comment:
What's the justification for removing block length notifications in seeds? I see no problem with being warned about a long block in this location.